test-requirements.txt

```diff
@@ -2,12 +2,14 @@
 # of appearance. Changing the order has an impact on the overall integration
 # process, which may cause wedges in the gate later.
 
+mock
 hacking<0.12,>=0.11.0 # Apache-2.0
 pytest
 pytest-cover
+ansible>=2.1.1
```

Comment:
remove this, make it part of a new PR for the ansible related work (since we need to add more details related to valid shell commands etc)